server/node.go

```diff
@@ -390,6 +402,94 @@ func (n *Node) gossipCapacities() {
 	})
 }
 
+// storesScanner will walk through all the stores in the node every
+// ctx.ScanInterval and store the status in the db.
+func (n *Node) storesScanner(stopper *util.Stopper) {
+	stopper.RunWorker(func() {
+		start := time.Now()
```

Comment:
This is just an aggregation of the output of the range scanner. Perhaps instead of waiting for the range scanner to *maybe* finish, you could just add this as a callback for the range scanner?